### ISSUE TITLE
Web UI cannot modify header information on login/create account

### ISSUE THREAD

**Issue by jack-margeson** [BOINC/boinc#5692]
**Describe the bug**
When trying to either sign up for an account or log in through the Web UI, the user is redirected to the login_action.php page and the following errors are displayed in browser:
```
Deprecated: Creation of dynamic property BoincUser::$id is deprecated in /home/boincadm/projects/apriori/html/inc/db_conn.inc on line 121

...

Deprecated: Creation of dynamic property BoincUser::$email_addr_change_time is deprecated in /home/boincadm/projects/apriori/html/inc/db_conn.inc on line 121

Warning: Cannot modify header information - headers already sent by (output started at /home/boincadm/projects/apriori/html/inc/db_conn.inc:121) in /home/boincadm/projects/apriori/html/inc/util.inc on line 169

Warning: Cannot modify header information - headers already sent by (output started at /home/boincadm/projects/apriori/html/inc/db_conn.inc:121) in /home/boincadm/projects/apriori/html/user/login_action.php on line 71
```

Additionally, the two following errors can be seen in the Apache2 log:
```
[Sat Jul 13 10:11:21.009853 2024] [php:warn] [pid 21202] [client *ip*] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/boincadm/projects/apriori/html/inc/db_conn.inc:121) in /home/boincadm/projects/apriori/html/inc/util.inc on line 169, referer: https://apriori.group/apriori/login_form.php

[Sat Jul 13 10:11:21.009881 2024] [php:warn] [pid 21202] [client *ip*] PHP Warning:  Cannot modify header information - headers already sent by (output started at /home/boincadm/projects/apriori/html/inc/db_conn.inc:121) in /home/boincadm/projects/apriori/html/user/login_action.php on line 71, referer: https://apriori.group/apriori/login_form.php
```

**Steps To Reproduce**
1. Create a new BOINC project using the latest code base from the master branch.
2. Sign SSL certification for the web UI and start the Apache service once configured.
3. Try creating a user or logging into an already created account via the Web UI.

**Expected behavior**
User should be able to log in/create an account. While creating an account ***does*** actually create a database entry for the user, the user is not directed to the correct page after creation.

**Screenshots**
n/a

**System Information**
 - OS: Ubuntu 24.04 LTS
 - BOINC Version: 1.4.2, latest server code (from master branch)

**Additional context**
I suspect this has something to do with how login information is passed between the user's browser and the server. I have not had a problem with this login flow under a HTTP implementation--but I am seeing this in this stock HTTPS implementation of the web UI.

**Comment by AenBleidd** [comment]
@davidpanderson, could you please take a look?

**Comment by davidpanderson** [comment]
I can't repro this on Ubuntu 18.04 using MariaDB 15.1.
Maybe it has to do with DB server.
What are you running (mysql --version)?

**Comment by jack-margeson** [comment]
mysql --version:
`mysql  Ver 8.0.37-0ubuntu0.24.04.1 for Linux on x86_64 ((Ubuntu))`

**Comment by davidpanderson** [comment]
Also, what PHP version?

**Comment by jack-margeson** [comment]
PHP:
```
PHP 8.3.6 (cli) (built: Jun 13 2024 15:23:20) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.3.6, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.6, Copyright (c), by Zend Technologies
```

Apache2:
```
Server version: Apache/2.4.58 (Ubuntu)
Server built:   2024-07-11T14:41:54
```

**Comment by davidpanderson** [comment]
#5693 works with PHP 8.2; hopefully with 8.3 as well

**Comment by jack-margeson** [comment]
Pulled down the changes locally and it looks like this fixes the issue (PHP ver 8.3 confirmed)--massive thanks for the quick response time and help!
